aws/cloudfront.tf

```diff
@@ -45,7 +45,7 @@ resource "aws_cloudfront_distribution" "richie_cloudfront_distribution" {
     target_origin_id = local.s3_static_origin_id
 
     forwarded_values {
-      query_string = false
+      query_string = true
```

Comment:
According to [cloudfront documentation](https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_ForwardedValues.html#cloudfront-Type-ForwardedValues-QueryString), this is deprecated and you should use an origin request policy instead.

Reply:
Apparently this is not yet merged into the aws terraform provider ( https://github.com/terraform-providers/terraform-provider-aws/issues/14373 ).

Reply:
Good catch. Let's upgrade to 0.13 first.